test/bench/memtier.test.lua

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env tarantool
+
+local tap = require('tap')
+local fio = require('fio')
+local has_popen, popen = pcall(require, 'popen')
+package.cpath = './?.so;' .. package.cpath
+local memcached = require('memcached')
+local test = tap.test('memcached benchmarks')
+
+if not has_popen then
+    test:plan(0)
+    os.exit(0)
+end
+
+test:plan(3)
+
+local is_test_run = os.getenv('LISTEN')
+
+if type(box.cfg) == 'function' then
+    box.cfg{
+        wal_mode = 'none',
+        memtx_memory = 100 * 1024 * 1024,
+    }
+    box.schema.user.grant('guest', 'read,write,execute', 'universe')
+end
+
+local port = 11211
+local mc = memcached.create('memcached', tostring(port), {})
+
+local function run_memtier(instance, memtier_path, proto)
+    instance:cfg({
+        protocol = proto,
+    })
+    local memtier_proto = 'memcache_' .. proto
+    local memtier_argv = {
+        memtier_path,
+        '--server=127.0.0.1',
+        string.format('--port=%d', port),
+        string.format('--protocol=%s', memtier_proto),
+        '--threads=10',
+        '--test-time=2',
+        '--hide-histogram'
+    }
+
+    local stdout = is_test_run and popen.opts.PIPE or popen.opts.INHERIT
+    local stderr = is_test_run and popen.opts.PIPE or popen.opts.INHERIT
```

Comment:
`PIPE` here is to suppress the output? It is better to use `DEVNULL` instead. Otherwise, if the pipe buffer will overflow, the child process may stuck on `write(1 or 2, <..>)`.

Reply:
(I know, I'm late. But I was curious, looked around and found something to highlight. I guess this is not a big problem, since you didn't meet it neither locally, nor in CI, so feel free to ignore.)

Reply:
> `PIPE` here is to suppress the output? It is better to use `DEVNULL` instead. 

AFAIR yes. Fixed in PR  https://github.com/tarantool/memcached/pull/99